src/utils/vscode/summarizeCodeContext.ts

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+import { backendURL } from "../../constants";
+import analyticsReporter from "../vscode/reporter";
+
+export default async function summarizeCodeConext({
+    pr_title,
+    pr_body,
+    block_of_code,
+  }: {
+    pr_title: string;
+    pr_body: string;
+    block_of_code: string;
+  }) {
+    const codeContextSummary = await axios
+    .post(`${backendURL}/api/openai/summarizeCodeContext`, {
+    pr_title,
+      pr_body,
```

Comment:
So this endpoint does not validate the user at all?

Reply:
What's the reason why it needs to?